CriticalMass/RequestManager.swift

```diff
@@ -7,51 +7,78 @@
 
 import os.log
 import UIKit
+import Network
 
 public class RequestManager {
-    private struct SendLocationPostBody: Codable {
-        var device: String
-        var location: Location
-    }
-
     private struct SendMessagePostBody: Codable {
         var device: String
         var messages: [SendChatMessage]
     }
 
     private let endpoint: URL
 
-    private var hasActiveRequest = false
```

Comment:
👍 Real like that we don't need that anymore :)

Reply:
I think the same way :)